pywal/export.py

```diff
@@ -25,7 +25,7 @@ def template(colors, input_file, output_file=None):
             # Color to be modified copied into new one
             new_color = util.Color(colors[cname].hex_color)
             # Execute each function to be done
-            for func in filter(None, funcs.split(")")):
+            for func in filter(None, re.split("\)|\.", funcs)):
```

Comment:
This line fails the CI tests because of invalid escape sequences. This can be fixed by escaping the backslashes (`"\\)|\\."`), or using a raw string literal (`r"\)|\."`)